src/backend/gl/src/factory.rs

```diff
@@ -148,7 +148,7 @@ impl Factory {
         else {
             let usage = match info.usage {
                 Data => gl::STATIC_DRAW,
```

Comment:
I don't see any way to know if the data is going to change often with the new usage, should we reintroduce the `Dynamic` variant then ?